Index: exo.ws.frameworks.servlet/src/main/java/org/exoplatform/ws/frameworks/servlet/StandaloneContainerInitializedListener.java
===================================================================
--- exo.ws.frameworks.servlet/src/main/java/org/exoplatform/ws/frameworks/servlet/StandaloneContainerInitializedListener.java	(revision 4236)
+++ exo.ws.frameworks.servlet/src/main/java/org/exoplatform/ws/frameworks/servlet/StandaloneContainerInitializedListener.java	(working copy)
@@ -80,13 +80,25 @@
          LOG.error("Error of configurationURL read", e);
       }
 
+      //  If no configuration in web.xml check system property.
+      if (configurationURL == null)
+         configurationURL = System.getProperty(CONF_URL_PARAMETER);
+
       try
       {
          StandaloneContainer.addConfigurationURL(configurationURL);
       }
       catch (MalformedURLException e)
       {
-         LOG.error("Error of addConfigurationURL", e);
+         // Try to use path, we do not need have full path (file:/path/conf) to configuration. Any relative path is OK.
+         try
+         {
+            StandaloneContainer.addConfigurationPath(configurationURL);
+         }
+         catch (MalformedURLException e2)
+         {
+            LOG.error("Error of addConfiguration", e2);
+         }
       }
 
       try
